src/libraries/System.Runtime/tests/System/ObjectDisposedExceptionTests.cs

```diff
@@ -42,5 +42,19 @@ public static void Ctor_String_String()
             Assert.Contains(message, exception.Message);
             Assert.Contains(objectName, exception.Message);
         }
+
+        [Fact]
+        public static void Throw_Object()
+        {
+            var obj = new object();
+            AssertExtensions.Throws<ObjectDisposedException>(() => ObjectDisposedException.Throw(obj));
```

Comment:
We should test that the thrown exception has the ObjectName property set the way we expect.

In this case

```suggestion
            ObjectDisposedException ex = AssertExtensions.Throws<ObjectDisposedException>(
                () => ObjectDisposedException.Throw(obj));
            
            Assert.Equals("System.Object", ex.ObjectName);
```

Then it'd be good to use some variety.